src/app/qgisapp.cpp

```diff
@@ -2651,7 +2651,7 @@ void QgisApp::dataSourceManager( const QString &pageName )
         case Qgis::LayerType::Plugin:
         case Qgis::LayerType::Annotation:
         case Qgis::LayerType::Group:
-          break;
+        case Qgis::LayerType::TiledMesh:           break;
```

Comment:
this is present in https://github.com/qgis/QGIS/pull/53630

Reply:
weird auto style from my IDE when saving...